packages/server/src/games/quiz.ts

```diff
@@ -21,6 +21,8 @@ const enum GameEvent {
   RoundCounter = 'roundCounter',
 }
 
+const GAME_MAX_GUESS = 4;
```

Comment:
Do we want to put this value as an attribute of Round?
This way, instead of watching against this const, we check against `this.currentRound.maxGuesses`. That will allow the `YES/NO` questions to be easily set with a max guess of one

Reply:
Yes you're right, I create an issue to create this attribute:
https://github.com/sQuiz-2/Core/issues/11

Reply:
We can either take care of this as part of this PR, or I can take care of it once this PR is merged

Reply:
I think it's better to resolve #11 in another PR, this one is just the begining of the idea and the logic.